js/build.js

```diff
@@ -25,7 +25,7 @@
     const testDataObject = {};
     let compiledRecordTemplate;
 
-    let recordTemplate = $('<div></div>').append($($recordTemplate.html() || '').find('fl-prop[data-path]').each(function(i, el) {
+    let recordTemplate = $('<div></div>').html($recordTemplate.html() || '').find('fl-prop[data-path]').each(function(i, el) {
```

Comment:
@tonytlwu, `recordTemplate` is never reassigned

```suggestion
    const recordTemplate = $('<div></div>').html($recordTemplate.html() || '').find('fl-prop[data-path]').each(function(i, el) {
```